test/acceptance/features/steps/kubernetesapi.py

```diff
@@ -23,6 +23,22 @@ def on_primaza_cluster_check_state(context, cluster, ce_name, state, timeout=60)
         timeout=timeout)
 
 
+@step(u'On Primaza Cluster "{cluster}", ClusterEnvironment "{ce_name}" is deleted')
```

Comment:
No need to set timeout as a parameter

Reply:
okay

Reply:
As we are not passing the timeout from the steps, Instead of setting it as a parameter in the function we can  have timeout=60 defined in the polling method.